sycl/test/lit.cfg.py

```diff
@@ -112,15 +112,15 @@
 if triple == 'nvptx64-nvidia-cuda':
     config.available_features.add('cuda')
 
+
 if triple == 'amdgcn-amd-amdhsa':
     config.available_features.add('hip_amd')
-    # For AMD the specific GPU has to be specified with --offload-arch
-    if not re.match('.*--offload-arch.*', config.sycl_clang_extra_flags):
-        raise Exception("Error: missing --offload-arch flag when trying to "  \
-                        "run lit tests for AMD GPU, please add "              \
-                        "--hip-amd-arch=<target> to buildbot/configure.py or add"  \
-                        "`-Xsycl-target-backend=amdgcn-amd-amdhsa --offload-arch=<target>` to " \
-                        "the CMake variable SYCL_CLANG_EXTRA_FLAGS")
+    # For AMD the specific GPU has to be specified with --offload-arch, only
+    # compiler tests are run so hardcode the offload arch to gfx908
```

Comment:
The comment says "gfx908", but code uses "gfx906". Typo?
```suggestion
    # compiler tests are run so hardcode the offload arch to gfx906
```